extension/vscode-howdoi/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "howdoi",
+  "displayName": "howdoi",
+  "publisher": "Diana",
+  "description": "Receive instant coding answers via a code editor.",
+  "version": "0.0.1",
+  "engines": {
+    "vscode": "^1.46.0"
+  },
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onCommand:howdoi.extension"
+  ],
+  "main": "./out/extension.js",
+  "contributes": {
+    "commands": [
+      {
+        "command": "howdoi.extension",
+        "title": "howdoi"
+      }
+    ]
+  },
+  "scripts": {
+    "copy": "ncp ../code-editor-integration/ src/code-editor-integration/",
+    "clean": "rimraf ./src/code-editor-integration/node_modules",
```

Comment:
Cool, I didn't know about `rimraf`.

Reply:
I removed `code-editor-integration/node_modules` from the `vscode` files since it was producing an error about file duplication when testing the `vscode` files.